test/parallel/test-tls-async-cb-after-socket-end.js

```diff
@@ -12,8 +13,8 @@ const tls = require('tls');
 
 const options = {
   secureOptions: SSL_OP_NO_TICKET,
-  key: fs.readFileSync(path.join(common.fixturesDir, 'test_key.pem')),
-  cert: fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'))
+  key: fs.readFileSync(path.join(fixtures.fixturesDir, 'test_key.pem')),
```

Comment:
This could be cleaned up further by making use of [`fixtures.readSync()`](https://github.com/nodejs/node/tree/master/test/common#fixturesreadsyncargs-enc) to replace the `fs.readFileSuync` _and_ the `path.join`!

Something like `key: fixtures.readSync('test_key.pem'),`

Reply:
Ahh thanks for pointing that out, will make changes now.